src/Microsoft.OData.Core/ODataDeltaResourceSet.cs

```diff
@@ -4,32 +4,14 @@
 // </copyright>
 //---------------------------------------------------------------------
 
+using Microsoft.OData.Metadata;

```

Comment:
Is this still needed?

Reply:
nope.

---
In reply to: [150156980](https://github.com/OData/odata.net/pull/989#discussion_r150156980) [](ancestors = 150156980)